pkgs/top-level/python-packages.nix

```
@@ -9108,6 +9108,8 @@ self: super: with self; {
 
   nbconvert = callPackage ../development/python-modules/nbconvert { };
 
+  nbdb = callPackage ../development/python-modules/nbdb { };
```

```suggestion
  nonbloat-db = callPackage ../development/python-modules/nonbloat-db { };
```
Shouldn't this be called liek this?